packages/o/openssl/xmake.lua

```diff
@@ -83,7 +102,13 @@ package("openssl")
             table.insert(configs, "no-makedepend")
             table.insert(configs, "/FS")
         end
-        os.vrunv("perl", configs)
+        import("lib.detect.find_tool")
+        local perl = assert(find_tool("perl", {paths={"$(env PERL)"}}), "package(openssl): perl not found!")
```

Comment:
we should not call `find_tool`, we should use `add_deps` to bind perl path end and run `perl`